docs/guides/1.19.0-upgrade-guide.md

```diff
@@ -0,0 +1,26 @@
+---
+page_title: "Upgrade Guide 1.19.0"
```

Comment:
As mentioned by Zuhair we can add a small note on the publishing of new terraform modules. Not a provider feature precisely, but we can mention as a side note.

Reply:
good point, let me add

Reply:
added